docs/user/pings/index.md

```diff
@@ -79,6 +79,9 @@ See [Using the Experiments API](../experiments-api.md) on how to record experime
 ## Ping submission
 
 The pings that the Glean SDK generates are submitted to the Mozilla servers at specific paths, in order to provide additional metadata without the need to unpack the ping payload.
+
+> **Note**: To keep resource usage in check, the Glean SDK allows only up to 10 ping submissions every 60 seconds. There are no exposed methods to change these rate limiting defaults.
```

Comment:
> There are no exposed methods to change these rate limiting defaults.

Do you believe methods to change this will be immediately necessary? I think maybe for testing purposes they might be? If so I can file a bug and link to it in the internal and user facing documentations.

Reply:
The long(er) term plan is to allow products to configure their own limits. Fenix might want tighter control compared to Firefox Desktop, for example. Let's file a bug about allowing products to customize the policy and link it here.